id3v1_test.go

```diff
@@ -5,11 +5,12 @@
 package tag
 
 import (
+	"./id3v1_test"
```

Comment:
Update this so that it's not a relative path (and put it below the std lib imports).